site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -64,12 +64,7 @@ export const UsersPage: FC<{ children?: ReactNode }> = () => {
   } = usersState.context;
 
   const { updateUsers: canEditUsers, viewDeploymentValues } = usePermissions();
-  const [rolesState] = useMachine(siteRolesMachine, {
-    context: {
-      hasPermission: canEditUsers,
-    },
-  });
-  const { roles } = rolesState.context;
+  const rolesQuery = useRoles({ enabled: canEditUsers });
```

Comment:
```suggestion
  const rolesQuery = useRoles({ canEditUsers });
```

I kinda like that the old name described _why_ is was enabled. maybe we could even just call it `canEditUsers`?

Reply:
I used `enabled` to keep the same name as `react-query` but since it is going to change to use the factory functions I think there is no diff at the end.